Travis CI is not happy

# Description of problem

travis CI is failing

```
=== RUN   TestNewAgentClient

--- FAIL: TestNewAgentClient (1.01s)

	Error Trace:	client_test.go:114

			client_test.go:124

			client_test.go:134

	Error:      	Should be true

	Test:       	TestNewAgentClient

	Messages:   	expect err message: Invalid vsock port	got: parse vsock://100:bar: invalid port ":bar" after host

=== RUN   TestNewAgentClientWithYamux

--- FAIL: TestNewAgentClientWithYamux (1.10s)

	Error Trace:	client_test.go:114

			client_test.go:148

			client_test.go:156

	Error:      	Should be true

	Test:       	TestNewAgentClientWithYamux

	Messages:   	expect err message: Invalid vsock port	got: parse vsock://100:bar: invalid port ":bar" after host

FAIL
```

--- a/.ci/install_go.sh
+++ b/.ci/install_go.sh
@@ -11,6 +11,12 @@ source "${cidir}/lib.sh"
 
 clone_tests_repo
 
+new_goroot=/usr/local/go
+
 pushd "${tests_repo_dir}"
-.ci/install_go.sh -p -f
+# Force overwrite the current version of golang
+[ -z "${GOROOT}" ] || rm -rf "${GOROOT}"
+.ci/install_go.sh -p -f -d "$(dirname ${new_goroot})"
+[ -z "${GOROOT}" ] || sudo ln -sf "${new_goroot}" "${GOROOT}"
+go version
 popd

--- a/.travis.yml
+++ b/.travis.yml
@@ -18,13 +18,13 @@ env:
   - target_branch=$TRAVIS_BRANCH
 
 before_script:
-  - ".ci/install_go.sh"
   - ".ci/static-checks.sh"
 
 before_install:
   - sudo apt-get update -qq
   - sudo apt-get install -y -qq automake
   - sudo apt-get install -y moreutils
+  - ".ci/install_go.sh"
 
 install:
   - cd ${TRAVIS_BUILD_DIR} && make

